docs/docs/cmd/spo/listitem/listitem-get.md

```diff
@@ -35,9 +35,14 @@ Get an item with ID _147_ from list with title _Demo List_ in site _https://cont
 m365 spo listitem get --listTitle "Demo List" --id 147 --webUrl https://contoso.sharepoint.com/sites/project-x
 ```
 
-
-Get an items Title and Created column and with ID _147_ from list with title _Demo List_ in site _https://contoso.sharepoint.com/sites/project-x_
+Get an items _Title_ and _Created_ column with ID _147_ from list with title _Demo List_ in site _https://contoso.sharepoint.com/sites/project-x_
 
 ```sh
 m365 spo listitem get --listTitle "Demo List" --id 147 --webUrl https://contoso.sharepoint.com/sites/project-x --properties "Title,Created"
 ```
+
+Get an items _Title_, _Created_ column and lookup column _Company_ with ID _147_ from list with title _Demo List_ in site _https://contoso.sharepoint.com/sites/project-x_
```

Comment:
I would also add this info to remarks section. That if the user would like to specify a command in properties which is a lookup type, she or he will need to also define which columns from the related list should be returned